app/views/projects/reports.html.erb

```diff
@@ -2,69 +2,78 @@
   <%= render :partial => 'projects/nav', :locals => {:project => @project} %>
 <% end %>
 <% content_for :title_bar do %>
-<% end %>
-
-<% cache [@project, params[:since]], expires: 2.days do %>
-
-<% unless since_options.empty? %>
-<div class="row">
-  <div class="col-sm-12 col-md-12 text-right">
-    |
+  <div class="project_nav">
+    <%= link_to t('Back'), project_path(@project), class: "btn btn-primary btn-sm" %>
     <% since_options.each do |qty| %>
       <% if params[:since] == qty.to_s %>
-        <%= t("projects.reports.n months ago", count: qty) %> |
+        <%= link_to t("projects.reports.n months ago", count: qty), "#", class: "btn btn-link btn-sm" %>
       <% else %>
-        <%= link_to t("projects.reports.n months ago", count: qty), reports_project_path(@project, since: qty) %> |
+        <%= link_to t("projects.reports.n months ago", count: qty), reports_project_path(@project, since: qty), class: "btn btn-default btn-sm" %>
       <% end %>
     <% end %>
     <% if params[:since].present? %>
-      <%= link_to t('projects.reports.all_time'), reports_project_path(@project) %> |
+      <%= link_to t('projects.reports.all_time'), reports_project_path(@project), class: "btn btn-default btn-sm" %>
     <% end %>
   </div>
+<% end %>
+
+<% cache [@project, params[:since]], expires: 2.days do %>
+
+<% unless since_options.empty? %>
+<div class="row">
+  <div class="col-sm-12 col-md-12 text-right">
+
+  </div>
 </div>
 <% end %>
 
 <div class="row">
   <div class="col-sm-12 col-md-12">
-    <h2><%= t('projects.reports.current_iteration') %> <small>(<%= current_iteration_start_date %> - <%= current_iteration_end_date %>)</small></h2>
 
-    <p>This iteration has <%= current_iteration_points %> points planned in <%= current_iteration_count %> stories (including <%= current_iteration_non_estimable %> bugs or chores).</p>
+    <div class="col-sm-6 col-md-6">
+      <h3><%= t('projects.reports.current_iteration') %> <small>(<%= current_iteration_start_date %> - <%= current_iteration_end_date %>)</small></h3>
 
-    <p>There are <%= accepted_points %> points accepted so far, which is <%= accepted_rate %> of the total.</p>
+      <p>This iteration has <%= current_iteration_points %> points planned in <%= current_iteration_count %> stories (including <%= current_iteration_non_estimable %> bugs or chores).</p>
```

Comment:
Should, but the phrase formation depends on several <%= %> which will be very tricky to compose in all the different languages. This part and the activity presenter are still missing proper I18n for now